docs/docsite/rst/user_guide/windows_usage.rst

```diff
@@ -418,11 +416,16 @@ Here are some examples on how to write Windows paths:
     tempdir: 'C:\\Windows\\Temp'
     tempdir: C:/Windows/Temp
 
+This is an example which will fail:
+
+.. code-block:: text
+
     # FAILS
     tempdir: "C:\Windows\Temp"
```

Comment:
This part broke the example: it is not valid YAML. Because this error is intentional, I broke up the example into three parts: two correct YAML+Jinja2, and this one formatted as `text` inbetween.